pack/df.json

```diff
@@ -672,7 +672,7 @@
         "deck_limit": 3,
         "faction_code": "haas-bioroid",
         "faction_cost": 3,
-        "flavor": "The Net abhors a vacuum. Any unexplained gap must be made.",
+        "flavor": "The Net abhors a vacuum. Any unexplained gap must be <strong>made.</strong>",
```

Comment:
Won't using `<strong>` here bold the text, rather than un-italicise it like on the card image?

For example, check QPM:

https://github.com/Alsciende/netrunner-cards-json/blob/master/pack/dad.json#L114